zio-http/src/main/scala/zhttp/service/Server.scala

```diff
@@ -294,6 +311,10 @@ object Server {
 
   private final case class RequestDecompression(enabled: Boolean, strict: Boolean) extends UServer
 
+  private final case class LowLevelLogging(logLevel: LogLevel) extends UServer
+
+  private final case class CustomLogger(logger: Logger) extends UServer
+
```

Comment:
Logging at the moment should only be used internally by contributors while they are debugging a problem. 
It's too early to share a public API for users that allows them to configure it. This increases our API's surface areas and will come in the way of producing a stable release.

Reply:
Thinking more about this it feels like this might be a usecase. For eg: We could implement #1222 by changing the default log level to `info`. Which can be turned completely off in production.

Reply:
done.